src/JPetGeantEventInformation.cpp

```diff
@@ -19,6 +20,10 @@ void JPetGeantEventInformation::Clear()
     fPromptLifetime = 0.;
     fVtxPosition.SetXYZ(0,0,0);
     fVtxPromptPosition.SetXYZ(0,0,0);
+    fMomentumGamma0.SetXYZ(0,0,0);
+    fMomentumGamma1.SetXYZ(0,0,0);
+    fMomentumGamma2.SetXYZ(0,0,0);
+    fMomentumGamma3.SetXYZ(0,0,0);
```

Comment:
all this initialization in constructor should be in *.h file

Reply:
This part of code corresponds to Clear function, not initialization